lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -133,6 +133,17 @@ export abstract class BaseManagedIdentitySource {
                 managedIdentityId
             );
 
+        // if claims are present, the MSI will get a new token
+        if (managedIdentityRequest.claims) {
```

Comment:
can we add logs to these two conditions? i.e. when claims or capabilities are present?